src/recommendpolicy/helperFunctions.go

```diff
@@ -73,6 +74,21 @@ func generatePolicy(name, namespace string, labels LabelMap) ([]types.KnoxSystem
 				return []types.KnoxSystemPolicy{}, err
 			}
 			policies = append(policies, policy)
+		} else if ms.Kind == types.KindKubeArmorHostPolicy && cfg.GetRecommendHost() {
+			client := cluster.ConnectK8sClient()
+			nodeList, err := client.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
```

Comment:
Ant client go APIs/watcher/listeners, please take in under customResourceHandler.go and call the API from there.